core/GlobalState.cc

```diff
@@ -1415,7 +1415,8 @@ void GlobalState::_error(unique_ptr<Error> error) const {
     }
     auto loc = error->loc;
     if (loc.file().exists() && error->what != errors::Infer::SuggestTyped &&
-        error->what != core::errors::Resolver::SigInFileWithoutSigil) {
+        error->what != core::errors::Resolver::SigInFileWithoutSigil &&
+        error->what != core::errors::Namer::MultipleBehaviorDefs) {
```

Comment:
Can we check for stripe mode here too to make it work correctly in both modes?

Reply:
Ah sure, but I am not too sure how to pass the `stripeMode` flag into the `GlobalState` object. Let me look around.

Reply:
Ok, I moved the definition of which error types should be ignored out of the `GlobalState` class. This should be cleaner now.

Reply:
I am open to better naming on the newly added methods/fields.